scripts/concatenate_files.py

```diff
@@ -0,0 +1,69 @@
+import json
+import os
+import re
+import yaml
+
+docs_root = "./docs"
+
+
+def concat(of, level, doc_path):
+    full_path = f"{docs_root}/{doc_path}.md"
+    if not os.path.exists(full_path):
+        full_path = f"{docs_root}/{doc_path}/index.md"
+
+    with open(full_path, "r") as doc_file:
+        doc_text = doc_file.read()
+        # parse YAML header
+        yaml_match = re.match("^---$((.|\n)*?)^---$", doc_text, flags=re.MULTILINE).groups(1)
+        if yaml_match:
+            header = yaml.safe_load(yaml_match[0])
+            of.write(f"""{"#"*level} {header["title"]}""")
+            # strip the YAML header
+            doc_text = re.sub("^---$((.|\n)*)?^---$", "", doc_text, flags=re.MULTILINE)
+        # use relative path for images
+        doc_text = doc_text.replace("](/images", "](images")
+        of.write(doc_text)
+
+
+f = open("./_data/menu_docs_dev.json", "r")
```

Comment:
```suggestion
f = open("./_data/menu_docs_dev.json")
```

Read is the default here. And perhaps use the with statement instead of leaving the file open?

Reply:
implemented these ✅